### PR TITLE
pt_PT Translations

### PR DESCRIPTION
**Pull request by Blackspirits**

### CODE REVIEW

File: _locales/pt_PT/messages.json [hunk 1]
```diff
@@ -1,20 +1,25 @@
 {
     "appNameNewTab": {
-        "message": "DuckieTV - Modo 'New Tab' "
+        "message": "DuckieTV - Modo 'Nova Aba'"
```
Comment (garfield69) [inline _locales/pt_PT/messages.json]: you have left out a comma at the end of the message parameter, which is required before the description parm starts.

File: _locales/pt_PT/messages.json [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
     },
     "appShortNameNewTab": {
-        "message": "DuckieTV "
+        "message": "DuckieTV"
+        "description": "O título abreviado de 'Nova Aba', para ser usado na pesquisa e outros."
     },
     "appDescNewTab": {
-        "message": "Tudo o que você precisa saber quando você vem favorito Series TV internacional em "
+        "message": "Faz a gestão das tuas séries e filmes favoritos a partir da nova aba do teu navegador",
+		"description": "Etiqueta da loja web para a versão de 'Nova Aba'."
```
Comment (garfield69) [inline _locales/pt_PT/messages.json]: could you sort out the indentation, lining up the start of the description with the start of the message parm above.
thanks.

**Comment by Blackspirits** [comment]
Done

**Comment by garfield69** [comment]
Cheers.

**Pull request merged**